kroxylicious-multitenant/src/test/resources/io/kroxylicious/proxy/filter/multitenant/OffsetFetch.test.yaml

```diff
@@ -15,13 +15,16 @@
               partitionIndexes: [0]
       requireStable: false
     diff:
+      - op: replace
+        path: "/groups/0/groupId"
+        value: tenant1-mygroup
       - op: replace
         path: "/groups/0/topics/0/name"
         value: tenant1-foo
   response:
     payload:
       groups:
-        - groupId: mygroup
+        - groupId: tenant1-mygroup
```

Comment:
It crossed my mind the other day should the tenant prefix be be `__<ID>__` I think Kafka already uses `__` to denote internal names.

Reply:
Let's get to that later :)